src/SDKs/Authorization/Management.Authorization/Microsoft.Azure.Management.Authorization.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<Description>Provides capabilities to query Microsoft Azure Management Authorization.</Description>
 		<AssemblyTitle>Authorization Library for Microsoft Azure Management Authorization</AssemblyTitle>
-		<Version>2.6.0-preview</Version>
+		<Version>2.7.0-preview</Version>
```

Comment:
@darshanhs90 missing < pacakgerelesenotes >

Reply:
@shahabhijeet  done,added it